docs/userguide/networkingcontainers.md

```diff
@@ -0,0 +1,224 @@
+            "OOMKilled": false,
+      ...
+
+Container names must be unique. That means you can only call one container
+`web`. If you want to re-use a container name you must delete the old container
+(with `docker rm`) before you can reuse the name with a new container. Go ahead and stop and them remove your `web` container.
+
+    $ docker stop web
+    web
+    $ docker rm web
+    web
+
+
+## Launch a container on the default network
+
+Docker includes support for networking containers through the use of **network drivers**.  By default, Docker provides two network drivers for you, the `bridge` and the `overlay` driver. You can also write a network driver plugin so that you can create your own drivers but that is an advanced task.
+
+Every installation of the Docker Engine automatically includes three default networks. You can list them:
+
+    $ docker network ls
+    NETWORK ID          NAME                DRIVER
+    18a2866682b8        none                null                
+    c288470c46f6        host                host                
+    7b369448dccb        bridge              bridge  
+
+The network named `bridge` is a special network. Unless you tell it otherwise, Docker always launches your containers in this network. Try this now:
+
+    $ docker run -itd --name=networktest ubuntu
+    74695c9cea6d9810718fddadc01a727a5dd3ce6a69d09752239736c030599741
+
+Inspecting the network is an easy way to find out the container's IP address.
+
+    [
+      {
+          "name": "bridge",
+          "id": "7b369448dccbf865d397c8d2be0cda7cf7edc6b0945f77d2529912ae917a0185",
+          "scope": "local",
+          "driver": "bridge",
+          "ipam": {
+              "driver": "default",
+              "config": [
+                  {
+                      "subnet": "172.17.0.1/16",
+                      "gateway": "172.17.0.1"
+                  }
+              ]
+          },
+          "containers": {
+              "74695c9cea6d9810718fddadc01a727a5dd3ce6a69d09752239736c030599741": {
+                  "endpoint": "3141aa83b33e60adae944d155fbd32a62e4b9eccc0893d8b34d6809877aa51dc",
+                  "mac_address": "02:42:ac:11:00:02",
+                  "ipv4_address": "172.17.0.2/16",
+                  "ipv6_address": ""
+              }
+          },
+          "options": {
+              "com.docker.network.bridge.default_bridge": "true",
+              "com.docker.network.bridge.enable_icc": "true",
+              "com.docker.network.bridge.enable_ip_masquerade": "true",
+              "com.docker.network.bridge.host_binding_ipv4": "0.0.0.0",
+              "com.docker.network.bridge.name": "docker0",
+              "com.docker.network.driver.mtu": "9001"
+          }
+      }
+    ]
+
+You can remove a container from a network by simply disconnecting the container. To do this, you supply both the network name and the container name. You can also container id.  In this example, tho, the name is faster.
```

Comment:
Drop "simply"

"You can also container id" -> "you can also use the container id"